bindings/python/quokka/utils.py

```diff
@@ -177,3 +183,36 @@ def parse_version(version: str) -> tuple[int, int, int]:
         )
 
     return parsed
+
+
+def find_reg_access(
```

Comment:
Maybe we can give it more explicit name than IDA API. ``find_register_access`` ?